### DISCUSSION TITLE
animated profile pictures/gif to be shown in-game client and browser

### DISCUSSION THREAD

**Discussion by V13R-31** [ppy/osu#24914] [Ideas]
Seeing your profile picture animated in-game, if you have set profile picture as animated/gif
also add a toggle for this, for some people who won't like it if there are

https://github.com/ppy/osu/assets/86092863/54cabb7b-5bd3-4c31-a120-902af0d46756

**Comment by Infiniti727** [comment]
Would also be neat if animated banners were supported. 

I've been using the same gif i made as my banner for years now because i cant buy supporter for my own to even change it, and since lazer only uses the first frame from the banners, i have to see this every time i see the result screen, check the profile, or just click on the top right corner to quickly check my current rank.
![image](https://github.com/user-attachments/assets/b635e039-94a6-4761-81e7-3790d89d3165)

Im tired of it, because it looks like its a badly made Loss meme.

I know this is most likely a me only thing and i could just somehow get supporter and change the gif so it starts with a different frame, but i still think it should have support to some extent.

If you are given the option to customize your profile with animated elements that work on the website, they should also work on Lazer, I think.
